### PR TITLE
added DDA line drawing code for c & c++

### PR DESCRIPTION
**Pull request by tusharsabhani**
Implemented DDA line algo in C & C++ 
need some time for implementing in java

### CODE REVIEW

**Comment by tusharsabhani** [comment]
I'll fix the indentation and would use descriptive variable names too but how to show sample output for a line, it can be only shown with the help of a image.

**Comment by jainaman224** [comment]
Please add link to image and host image somewhere.

**Comment by tusharsabhani** [comment]
added sample outputs

File: DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#include<stdio.h>
+#include<conio.h>
+#include<stdlib.h>
+#include<graphics.h>
+void main()
+{
+	float x1,x2,y1,y2,dx,dy,steps,xi,yi,i;
```
Comment (jainaman224) [inline DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: Please use 4 space indentation in all files.

Comment (tusharsabhani) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: added 4 space indentation 
now have a look once

Comment (jainaman224) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: Not fixed.

Comment (tusharsabhani) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: check now

File: DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    xi = dx / steps;
+    yi = dy / steps;
+
+    for(i=1; i<steps; i++)
+    {
+        putpixel(x1, y1, 15);
+        x1 = x1 + xi ;
+        y1 = y1 + yi;
+    }
+    getch();
```
Comment (jainaman224) [inline DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: I think getch() is included in conio library and not supported by standard gcc.

Comment (tusharsabhani) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c]: Fixed it

File: DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.cpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        x1 = x1 + xi ;
+        y1 = y1 + yi;
+    }
+    getch();
+    closegraph();
+
+}
+
+//Sample output - https://ibb.co/3mqM8MK
+
```
Comment (jainaman224) [inline DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.cpp]: Add only one new line at the end of file.

Comment (tusharsabhani) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.cpp]: Can you please remove the extra ones and approve it?

Comment (jainaman224) [reply DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.cpp]: I have done so. Please be careful from next time.

**Pull request merged**